nixos/modules/services/misc/paperless.nix

```diff
@@ -229,29 +227,11 @@ in
 
       This sets `OMP_NUM_THREADS` to `1` in order to mitigate the issue. See
       https://github.com/NixOS/nixpkgs/issues/240591 for more information
-    '' // mkOption { default = true; };
```

Comment:
this isn't expected, right, @Stunkymonkey?

Reply:
you are totally right, might be i did something wrong while rebasing.

I reverted all my changes and rerun the script.

Reply:
On this file only, I hope!